rsmtool/notebooks/skll_model.ipynb

```diff
@@ -14,29 +14,19 @@
     "collapsed": true
    },
    "outputs": [],
-   "source": [
-    "Markdown('Model used: **{}**'.format(model_name))"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "objective_used = skll_objective if skll_objective else 'neg_mean_squared_error'\n",
-    "Markdown('Tuning objective used: **{}**'.format(objective_used))"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
-   "source": [
-    "Markdown('Number of features in model: **{}**'.format(len(features_used)))"
+    "intro_markdown_str = f\"\"\"### Characteristics \n",
+    "\n",
+    "- Model: **{model_name}**\n",
+    "- Number of features: **{len(features_used)}**\n",
+    "- Tuning objective: **{objective_used}**\n",
+    "\"\"\"\n",
+    "\n",
+    "if skll_fixed_parameters:\n",
+    "    intro_markdown_str += f\"- Custom fixed parameters: **{skll_fixed_parameters}**\"\n",
```

Comment:
We *could* print the parameters as a bullet list to make the report look nicer but it is a nice to have.

Reply:
I agree that would look nicer. Let me see if that's an easy fix.

Reply:
Just pushed a commit to print as bulleted list. Try it out?

Reply:
Looks great to me - thank you!